pep-0427.txt

```diff
@@ -312,9 +312,9 @@ checker only needs to establish that RECORD matches the signature.
 
 See
 
-- https://datatracker.ietf.org/doc/html/rfc7515
+- :rfc:`7515`
```

Comment:
Hmm, I see. To make them more closely match, we could just tweak the other two instead, to something like

```rst
- `Internet-Draft: JSON Private Key <https://datatracker.ietf.org/doc/html/draft-jones-jose-json-private-key.html>`
```

To note, I-Ds are intentionally made to not be too friendly to easily reference, since you really aren't "supposed" to outside the context of an explicit citation as a work in progress (and the two mentioned are now expired).